### ISSUE TITLE
@ngtools upgrade breaks everything, webpack: Failed to compile

### ISSUE THREAD

**Issue by kgish** [PatrickJS/starter#1971]
Using `"@ngtools/webpack": "^1.10.0",` in my `package.json` file works fine, but when upgrading to `"@ngtools/webpack": "^1.10.1",` everything breaks.

I get many failures, starting like this:

```
Module not found: Error: Can't resolve '.../src/node_modules/@angular/common'
...
Module not found: Error: Can't resolve '.../src/node_modules/@angular/common/http'
...
Module not found: Error: Can't resolve '.../src/node_modules/@angular/core'
...
webpack: Failed to compile
```
By reverting back to `"@ngtools/webpack": "^1.10.0",` everything works fine again.

Is this a known issue?

Here's my `package.json` file:
```
{
...
  "dependencies": {
    "@angular/animations": "^5.2.6",
    "@angular/common": "^5.2.6",
    "@angular/compiler": "^5.2.6",
    "@angular/core": "^5.2.6",
    "@angular/forms": "^5.2.6",
    "@angular/http": "^5.2.6",
    "@angular/platform-browser": "^5.2.6",
    "@angular/platform-browser-dynamic": "^5.2.6",
    "@angular/router": "^5.2.0",
    "@biesbjerg/ngx-translate-extract": "^2.3.4",
    "@ngx-translate/core": "^9.1.1",
    "@ngx-translate/http-loader": "^2.0.1",
    "bootstrap": "^4.0.0",
    "classlist.js": "^1.1.20150312",
    "core-js": "^2.5.3",
    "http-server": "^0.11.1",
    "ie-shim": "^0.1.0",
    "jquery": "^3.3.1",
    "ng-recaptcha": "^3.0.3",
    "ngx-cookie-service": "^1.0.10",
    "popper.js": "^1.12.9",
    "reflect-metadata": "^0.1.12",
    "rxjs": "^5.5.6",
    "zone.js": "^0.8.20"
  },
  "devDependencies": {
    "@angular-devkit/build-optimizer": "^0.4.2",
    "@angular/cli": "~1.7.1",
    "@angular/compiler-cli": "^5.2.0",
    "@angular/language-service": "^5.2.0",
    "@compodoc/compodoc": "^1.0.7",
    "@cypress/webpack-preprocessor": "^1.1.3",
    "@ngtools/webpack": "^1.10.1",
    "@types/hammerjs": "^2.0.35",
    "@types/jasmine": "~2.8.3",
    "@types/jasmine-expect": "^3.8.1",
    "@types/jasminewd2": "~2.0.2",
    "@types/jquery": "^3.3.0",
    "@types/node": "^9.4.6",
    "@types/source-map": "^0.5.7",
    "@types/uglify-js": "^2.6.30",
    "@types/webpack": "^3.8.8",
    "add-asset-html-webpack-plugin": "^2.1.2",
    "angular2-template-loader": "^0.6.2",
    "awesome-typescript-loader": "^3.4.1",
    "codelyzer": "^4.0.1",
    "compression-webpack-plugin": "^1.1.7",
    "copy-webpack-plugin": "^4.4.1",
    "cross-env": "^5.1.3",
    "css-loader": "^0.28.9",
    "eslint": "^4.18.1",
    "eslint-config-standard": "^11.0.0",
    "eslint-plugin-angular": "^3.2.0",
    "eslint-plugin-cypress": "^2.0.1",
    "eslint-plugin-import": "^2.9.0",
    "eslint-plugin-node": "^6.0.0",
    "eslint-plugin-promise": "^3.6.0",
    "eslint-plugin-standard": "^3.0.1",
    "exports-loader": "^0.7.0",
    "expose-loader": "^0.7.4",
    "extract-text-webpack-plugin": "^3.0.2",
    "favicons-webpack-plugin": "^0.0.7",
    "file-loader": "^1.1.9",
    "find-root": "^1.1.0",
    "gh-pages": "^1.1.0",
    "html-webpack-plugin": "^2.30.1",
    "image-webpack-loader": "^4.1.0",
    "imports-loader": "^0.8.0",
    "inline-manifest-webpack-plugin": "^3.0.1",
    "istanbul-instrumenter-loader": "^3.0.0",
    "jasmine-core": "^2.99.1",
    "jasmine-expect": "^3.8.3",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~2.0.0",
    "karma-chrome-launcher": "^2.2.0",
    "karma-cli": "~1.0.1",
    "karma-coverage": "^1.1.1",
    "karma-coverage-istanbul-reporter": "^1.4.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "karma-mocha-reporter": "^2.2.5",
    "karma-remap-coverage": "^0.1.5",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-webpack": "^2.0.9",
    "modernizr": "^3.5.0",
    "ngc-webpack": "^4.1.2",
    "node-sass": "^4.7.2",
    "normalize-scss": "^7.0.1",
    "npm-check": "^5.5.2",
    "npm-run-all": "^4.1.2",
    "optimize-js-plugin": "^0.0.4",
    "parse5": "^4.0.0",
    "preload-webpack-plugin": "^2.2.0",
    "raw-loader": "^0.5.1",
    "rimraf": "^2.6.2",
    "sass-loader": "^6.0.6",
    "script-ext-html-webpack-plugin": "^1.8.8",
    "sonarqube-scanner": "^2.0.2",
    "source-map-loader": "^0.2.3",
    "string-replace-loader": "^1.3.0",
    "style-loader": "^0.20.2",
    "stylelint": "^9.1.1",
    "stylelint-config-sass-guidelines": "^4.1.0",
    "stylelint-order": "^0.8.1",
    "stylelint-scss": "^2.4.0",
    "stylelint-webpack-plugin": "^0.10.3",
    "svg-url-loader": "^2.3.1",
    "to-string-loader": "^1.1.5",
    "ts-node": "^5.0.0",
    "tslib": "^1.9.0",
    "tslint": "~5.9.1",
    "tslint-eslint-rules": "^5.0.0",
    "tslint-loader": "^3.5.3",
    "tslint-sonarts": "^1.4.0",
    "typedoc": "^0.10.0",
    "typescript": "^2.7.2",
    "uglifyjs-webpack-plugin": "^1.2.0",
    "url-loader": "^0.6.2",
    "webpack": "^3.11.0",
    "webpack-dev-middleware": "^2.0.5",
    "webpack-dev-server": "^2.11.1",
    "webpack-dll-bundles-plugin": "^1.0.0-beta.5",
    "webpack-merge": "~4.1.0",
    "webpack-stats-graph": "^1.0.8"
  },
...
  "resolutions": {
    "remap-istanbul": "0.9.6"
  }
}
```

**Comment by craigmatthew412** [comment]
I tried to narrow down which NPM module it was which was causing the issue for hours yesterday...this is also related to issue 1970.  If you change your node_modules folder to be relative to baseUrl property in tsconfig.json and tsconfig.webpack.json then you should be all set.  I have validated both scenarios work...using the existing path for @angular/* in tsconfig and using @ngtools/webpack 1.10.0 and also using relative path for @angular/* using @ngtools/webpack 1.10.1

@gdi2290 I will get around to submitting a PR later this morning for this once I get back to my desk

**Comment by legomanaustralia** [comment]
Seeing this also :(

**Comment by almothafar** [comment]
I think this should be fixed in PR #1975 
The solution for it already posted in #1974

**Comment by lukaszsamson** [comment]
I managed to get this to work with `@ngtools/webpack` 1.10.2 by changing
```
"paths": {
      "@angular/*": ["node_modules/@angular/*"]
    },
```
to
```
"paths": {
      "@angular/*": ["../node_modules/@angular/*"]
    },
```
in `tscongig.webpack.json`

**Comment by jialianwu** [comment]
Guys, I have the exact same issue, and I guess there is something wrong with the version of node (v.8x) and npm that I was using for Ubuntu 18.04.
Here is what I did that solve the problem:
1. Remove the folder: rm -r node_modules
2. If you don't have curl: -> sudo apt install curl
3. Add the new version to the library: -> curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -
4. sudo apt install nodejs  /*This will install npm as well
5. npm install /* that is it

I run ng serve again, and all the problems are gone. 
I still have the @ngtools/webpack in my package.json or package-lock.json.  
Above is everything I did for the fix. 
First time posting, hope this help~!
